src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/HttpCacheProxyCacheableRequest.java

```diff
@@ -125,7 +125,7 @@
         this.connectRouteId = connectRouteId;
         this.connectReplyId = connectReplyId;
         this.connectInitialId = connectInitialId;
-        this.initialWindow = factory.responseBufferPool.slotCapacity();
+        this.initialWindow = factory.writeBuffer.capacity();
```

Comment:
Why is `initialWindow` bounded by `writeBuffer.capacity()` now?
If the intent is to let as much `DATA` `payload` as possible be sent without exceeding the maximum size of a frame, then you would also need to account for the `DATA` framing itself, since `payload` is only part of the frame.